supply-chain/audits.toml

```diff
@@ -785,6 +809,17 @@ who = "Jack Grigg <jack@electriccoin.co>"
 criteria = "safe-to-deploy"
 delta = "0.4.8 -> 0.6.0"
 
+[[audits.schemars]]
+who = "Schell Carl Scivally <efsubenovex@gmail.com>"
+criteria = "safe-to-deploy"
+delta = "0.8.12 -> 0.9.0"
+
+[[audits.schemars]]
+who = "Schell Carl Scivally <efsubenovex@gmail.com>"
+criteria = "safe-to-deploy"
+delta = "0.9.0 -> 1.1.0"
+notes = "These changes improve security by forbidding unsafe code."
```

Comment:
As I noted above, this is actually reverting a change from a previous update that weakened the `unsafe_code` guarantee.

What were the other changes to generated code in this update?